doc/parameters/param_list.md

```diff
@@ -119,7 +119,8 @@ When `plugins` parameter is not overridden, the following default plugins are lo
 | `<data source>`.inf_is_valid | false | Are infinite returns from laser scanners valid measurements |
 | `<data source>`.marking | true | Whether source should mark in costmap |
 | `<data source>`.clearing | false | Whether source should raytrace clear in costmap |
-| `<data source>`.obstacle_range | 2.5 | Maximum range to mark obstacles in costmap |
+| `<data source>`.obstacle_max_range | 2.5 | Maximum range to mark obstacles in costmap |
+| `<data_source>`.obstacle_min_range | 0.0 | Minimum range to mark obstacles in costmap | 
```

Comment:
I thought we were going to do a min for both obstacle and ray trace? Then use the obstacle for the `marking` functions and the raytrace for the `clearing` functions - essentially decoupling them the same way the max ranges are

Reply:
I'm not sure I understand what you mean. I created seperate min variables for the obstacle and raytracing. When I created the obstacle_min_range variable, I realized I could rename the obstacle_range to obstacle_max_range to maintain consistency

Reply:
But it looks like you also removed the other one. There are 2 max range parameters now and should have also 2 analog min parameters

Reply:
No I have not removed the minimum range parameters